src/data/frameworks.ts

```diff
@@ -224,5 +299,8 @@ export const FrameworkIDs: Record<
     Mechanized: 'F9B8E96C71FF2E459C3D06BA70175D17',
     Neonband: '3E4350A85723C847A89B388C57655570',
     'Scale Brigade': '9B1D97F7686B8B4DB460D6CBAC18F92E',
+    Waster: 'NO_ID_SINCE_DIDNT_EXIST_IN_SEASON_1',
```

Comment:
These are weapon frameworks added in sseason 1. Since the new secondaries were introduced in season 2, they don't have this framework. I should maybe clarify this a bit more in a comment.

Reply:
Ah sorry, I meant framework when I said skin. Got it, so they don't exist in the game itself. I'm not suggesting you need to do this, but this field should probably just allow nullable values (or similar) in case something similar happens again.

This is totally fine as is though, I did understand what you meant with the comment.

Also, you've no need to wait for reviews or anything to merge PRs; they're just in place for standard branch protection.

Reply:
Yes, I thought about nullable values too since this is kinda ugly.

I am currently talking with other modders from the community to maybe obtain the image for the new armor paintjob which isn't currently available on the wiki. If I don't get it by tomorrow I will just merge this and deal with that armor paintjob at a later point.